commands/game/flee.js

```diff
@@ -4,53 +4,55 @@ const { ids } = require("../../config")
 const gamewarns = require("../../schemas/gamewarns")
 
 module.exports = {
-    name: "suicide",
-    description: "Kill yourself... In game! NOT IN REAL LIFE!",
-    usage: `${process.env.PREFIX}suicide [player]`,
+    name: "flee",
+    description: "Flee from the game!",
+    usage: `${process.env.PREFIX}flee [player]`,
     gameOnly: true,
     run: async (message, args, client) => {
         if (message.member.permissions.has("MANAGE_CHANNELS")) {
             if (args[0]) {
                 let guy = message.guild.members.cache.find((m) => m.nickname === args[0])
                 if (guy) {
                     let role = db.get(`role_${guy.id}`)
-                    db.set(`suicided_${guy.id}`, true)
+                    db.set(`fled_${guy.id}`, true)
```

Comment:
If you change this you also need to change it in line 33

Reply:
Also change it in:

https://github.com/wwosimulation/Narrator-bot/blob/0e9069c43affcef81671b9dff10c9097458e2019/commands/narrator/kill.js#L21